client/coral-framework/actions/config.js

```diff
@@ -31,3 +23,31 @@ export const updateConfiguration = newConfig => (dispatch, getState) => {
     })
     .catch(error => dispatch(updateConfigFailure(error)));
 };
+
+export const updateOpenStream = closedBody => (dispatch, getState) => {
+  const assetId = getState().items.get('assets')
+    .keySeq()
+    .toArray()[0];
+
+  dispatch(updateConfigRequest());
+
+  coralApi(`/asset/${assetId}/status`, {method: 'PUT', body: closedBody})
+    .then(() => {
+      dispatch(addNotification('success', lang.t('successUpdateSettings')));
+      dispatch(updateConfigSuccess(closedBody));
+    })
+    .catch(error => dispatch(updateConfigFailure(error)));
+};
+
+const openStream = () => ({type: actions.OPEN_COMMENTS});
+const closeStream = () => ({type: actions.CLOSE_COMMENTS});
+
+export const updateOpenStatus = status => dispatch => {
+  if (status === 'open') {
+    dispatch(openStream());
+    dispatch(updateOpenStream({closedAt: null}));
+  } else {
+    dispatch(closeStream());
+    dispatch(updateOpenStream({closedAt: new Date().getTime(), closedMessage: ''}));
```

Comment:
@okbel I think a null closed message means that the stream is open, while a date indicates the time at which it will close (`Date().getTime()` means close it now.)